apps/api/src/app/events/services/chat-service/handlers/linenotify.handler.ts

```diff
@@ -0,0 +1,18 @@
+import { ICredentials } from '@novu/dal';
+import { ChannelTypeEnum } from '@novu/stateless';
+import { BaseChatHandler } from './base.handler';
+import { LinenotifyProvider } from '@novu/linenotify';
```

Comment:
```suggestion
import { LineNotifyProvider } from '@novu/linenotify';
```
As the brand name seems to be LINE Notify, please follow this camelcasing in the code.